### ISSUE TITLE
Community Edition has no detach()?

### ISSUE THREAD

**Issue by allen3371** [jsplumb/jsPlumb#656]
https://jsplumbtoolkit.com/community/apidocs/classes/jsPlumb.html
i cant find the jsplumb.detach() function
but in document
https://jsplumbtoolkit.com/community/doc/removing.html
the doc have teh detach() function

**Comment by sporritt** [comment]
the "removing" page needs updating, check the changelog for 2.4.0

https://jsplumbtoolkit.com/community/doc/changelog.html

it's `deleteConnection` now
